event/generate.go

```diff
@@ -30,6 +30,7 @@ func main() {
 		"animationend":             "AnimationEnd",
 		"animationiteration":       "AnimationIteration",
 		"animationstart":           "AnimationStart",
+		"appinstalled":             "AppInstalled",
```

Comment:
I'm not sure if "App" should be considered a full word, or if it's short for "Application". If the former, this is good. If the latter, this should be mapped to "ApplicationInstalled", similar to how "dblclick" is mapped to "DoubleClick" below (rather than to "DblClick").

Given that the event is [described](https://developer.mozilla.org/en-US/docs/Web/Events/appinstalled) as (emphasis mine):

> The `appinstalled` event of the Web Manifest API is fired when the browser has successfully installed a page as an **application**.

I'm leaning slightly towards "ApplicationInstalled".

I'll defer to @slimsag on this.

Reply:
I agree, `ApplicationInstalled` is preferred here.